### DISCUSSION TITLE
USD to Coffee

### DISCUSSION THREAD

**Discussion by leeyemi** [tearust/teaproject#23] [General]
Hi, when click the detail on leader board found below "USD"
![image](https://user-images.githubusercontent.com/88189869/129710959-e6d2af71-19c7-47fc-8ba4-aae07975f859.png)

**Comment by kevingzhang** [comment]
Good catch, this issue has been assigned to the developer. thank you.
You are in the second position on the leaderboard now. You did a great job!

**Comment by liyangwood** [comment]
We fix it now. thanks for issue.
